crt_portal/cts_forms/templates/forms/complaint_view/show/index.html

```diff
@@ -14,7 +14,7 @@
           </div>
           <div class="display-flex flex-align-center details-id">
             <h2 class="margin-right-205 margin-top-0 margin-bottom-0">
-              ID: {{data.id}} 
+              ID: {{ data.public_id|slice:":-3" }}-{{ data.public_id|slice:"-3:" }}
```

Comment:
It may be worthwhile to include the `-` as part of the `public_id` itself. If that's how it's going to be displayed and referenced throughout the system, it feels like its functionally part of the ID.